### PR TITLE
Add peru prescription component

### PR DESCRIPTION
**Pull request by MiguelAHPpih**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/pihcore/config/Components.java [hunk 1]
```diff
@@ -67,6 +67,7 @@
         public static final String VISIT_NOTE = "visitNote";  // get rid of the left visits bar and enable form sections
         public static final String ALLERGIES = "allergies";
         public static final String PRESCRIPTIONS = "prescriptions";  // TODO no longer used?
+        public static final String PERU_PRESCRIPTIONS="peruPrescriptions";
```
Comment (mogoodrich) [inline api/src/main/java/org/openmrs/module/pihcore/config/Components.java:70]: I think we could just name this "Prescription":

**Review by mogoodrich — COMMENTED** [review]
LGTM, with the general suggestion that I think we can just name this "Prescription"

File: api/src/main/java/org/openmrs/module/pihcore/config/Components.java [hunk 2]
```diff
@@ -67,6 +67,7 @@
         public static final String VISIT_NOTE = "visitNote";  // get rid of the left visits bar and enable form sections
         public static final String ALLERGIES = "allergies";
         public static final String PRESCRIPTIONS = "prescriptions";  // TODO no longer used?
+        public static final String PRESCRIPTION="prescription"; //Used for Perú
```
Comment (brandones) [inline api/src/main/java/org/openmrs/module/pihcore/config/Components.java:70]: I hadn't noticed that we have an old component named `prescriptions`. We should clean that up.

**Review by brandones — APPROVED** [review]

File: api/src/main/java/org/openmrs/module/pihcore/config/Components.java (hunk 2 above)
Comment (MiguelAHPpih) [reply api/src/main/java/org/openmrs/module/pihcore/config/Components.java:70]: Do you want me to erase it or make into a comment?

Comment (brandones) [reply api/src/main/java/org/openmrs/module/pihcore/config/Components.java:70]: Oh, you don't need to do anything. We'd have to make sure it's not used anywhere, etc. I was just making a note to myself and @mogoodrich .

Comment (mogoodrich) [reply api/src/main/java/org/openmrs/module/pihcore/config/Components.java:70]: Ah, good catch @brandones ... I just went ahead and removed this... if we are wrong, and this is still used, I'd rather it fail fast and clean it up... having both a "prescription" and a "prescriptions" component would be confusing.

**Pull request merged**